test/helpers/spectron/index.ts

```diff
@@ -147,7 +145,8 @@ export function useSpectron(options: ITestRunnerOptions = {}) {
 
     // Pretty much all tests except for onboarding-specific
     // tests will want to skip this flow, so we do it automatically.
-    await t.context.app.client.waitForVisible('a=Setup later'); // wait for the loader disappearing
+
+    await t.context.app.client.waitForExist('.main-loading', 5000, true);
```

Comment:
does this basically mean we're waiting 5s after the main loader appears each time? or is it a timeout of 5s? if it's the former i feel like with how many times we start the app in our test suite this is going to take forever